AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/AppLovin/MRECs/Programmatic/ALDemoProgrammaticMRECViewController.m

```diff
@@ -31,39 +31,70 @@ - (void)viewDidAppear:(BOOL)animated
-                                 [NSLayoutConstraint constraintWithItem: self.adView
-                                                              attribute: NSLayoutAttributeWidth
-                                                              relatedBy: NSLayoutRelationEqual
-                                                                 toItem: nil
-                                                              attribute: NSLayoutAttributeNotAnAttribute
-                                                             multiplier: 1.0
-                                                               constant: kMRECWidth]]];
+    
+    [self.view addConstraints: @[
+        [NSLayoutConstraint constraintWithItem: self.callbackTableView
+                                     attribute: NSLayoutAttributeTop
+                                     relatedBy: NSLayoutRelationEqual
+                                        toItem: self.view
+                                     attribute: NSLayoutAttributeTop
+                                    multiplier: 1.0
+                                      constant: 0],
+        [NSLayoutConstraint constraintWithItem: self.callbackTableView
+                                     attribute: NSLayoutAttributeLeading
+                                     relatedBy: NSLayoutRelationEqual
+                                        toItem: self.view
+                                     attribute: NSLayoutAttributeLeading
+                                    multiplier: 1.0
+                                      constant: 0],
+        [NSLayoutConstraint constraintWithItem: self.callbackTableView
+                                     attribute: NSLayoutAttributeTrailing
+                                     relatedBy: NSLayoutRelationEqual
+                                        toItem: self.view
+                                     attribute: NSLayoutAttributeTrailing
+                                    multiplier: 1.0
+                                      constant: 0],
+        [NSLayoutConstraint constraintWithItem: self.adView
+                                     attribute: NSLayoutAttributeCenterX
+                                     relatedBy: NSLayoutRelationEqual
+                                        toItem: self.view
+                                     attribute: NSLayoutAttributeCenterX
+                                    multiplier: 1.0
+                                      constant: 0.0],
+        [NSLayoutConstraint constraintWithItem: self.adView
+                                     attribute: NSLayoutAttributeTop
+                                     relatedBy: NSLayoutRelationEqual
+                                        toItem: self.callbackTableView
+                                     attribute: NSLayoutAttributeBottom
+                                    multiplier: 1.0
+                                      constant: 10.0],
+        [NSLayoutConstraint constraintWithItem: self.view
```

Comment:
nit: the top and bottom margins for the `self.adView` should be equal (10.0 vs 20.0)

Reply:
ah ok the storyboard one was 10 so i kept it that